docs/topics/subcriptions_and_registrations.md

```diff
@@ -19,6 +44,34 @@ On the one hand, subscriptions:
 
 On the other hand, registrations:
 
+    POST /v1/registry/registerContext
```

Comment:
Are registrations also available for NGSIv2? If that's the case, I think it is better using v2.